src/panels/lovelace/components/hui-conditional-base.ts

```diff
@@ -0,0 +1,63 @@
+import { customElement, property, UpdatingElement } from "lit-element";
+
+import {
+  checkConditionsMet,
+  validateConditionalConfig,
+} from "../common/validate-condition";
+import { HomeAssistant } from "../../../types";
+import { LovelaceCard } from "../types";
+import { LovelaceRow, ConditionalRowConfig } from "../entity-rows/types";
+import { ConditionalCardConfig } from "../cards/types";
+
+@customElement("hui-conditional-base")
+export class HuiConditionalBase extends UpdatingElement {
+  @property() public hass?: HomeAssistant;
+  @property() protected _config?: ConditionalCardConfig | ConditionalRowConfig;
+  protected _element?: LovelaceCard | LovelaceRow;
+
+  public setConfig(config: ConditionalCardConfig | ConditionalRowConfig): void {
```

Comment:
Can we make this protected and give it another name, like `checkConfig`, and just call it instead of calling super?